leo/commands/add.rs

```diff
@@ -21,7 +21,12 @@
 //    leo add -a author -p package_name
 //
 
-use crate::{cli::CLI, cli_types::*, config::*, errors::AddError::*};
+use crate::{
+    cli::CLI,
+    cli_types::*,
+    config::*,
+    errors::{AddError::*, CLIError::AddError},
```

Comment:
Please adhere to the model we currently have for errors. CLIError should not explicitly define an AddError, you can derive this implicitly.